From 4b5b6208a78fb459b764eb451afb4d47f98cc143 Mon Sep 17 00:00:00 2001
From: Johannes Lorenz <j.git@lorenz-ho.me>
Date: Sat, 17 Jun 2023 21:39:22 +0200
Subject: [PATCH 1925/1936] Exclude port from port checker tests

This port was found buggy in CI tests, so we disable it for now. After
branch `wavetable` will have been merged, the port will change
significantly and might pass the tests.
---
 rtosc                  | 2 +-
 src/Synth/OscilGen.cpp | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/rtosc b/rtosc
index 6f151af0..1f801a87 160000
--- a/rtosc
+++ b/rtosc
@@ -1 +1 @@
-Subproject commit 6f151af012a659eb3cf2e61b6ffa3c62a4852897
+Subproject commit 1f801a8703c4274b7af13222b6f9858a8d32bf59
diff --git a/src/Synth/OscilGen.cpp b/src/Synth/OscilGen.cpp
index 4c498f4a..bc03b34f 100644
--- a/src/Synth/OscilGen.cpp
+++ b/src/Synth/OscilGen.cpp
@@ -159,6 +159,7 @@ const rtosc::Ports OscilGen::non_realtime_ports = {
             }
         }},
     {"basefuncFFTfreqs::b", rProp(parameter) rProp(non-realtime)
+        rProp(no port checker) // buggy port
         rBlobType(f) rDepends(magnitude, phase) rDefault([0.f 0.f ...])
         rDoc("FFT freqs of base function"),
         NULL, rBOIL_BEGIN
-- 
2.47.0

